File: Task/Josephus-problem/Python/josephus-problem-2.py
>>>def josephus(n, k):
        r = 0
        for i in xrange(1, n+1):
            r = (r+k)%i
        return 'Survivor: %d' %r

>>> print(josephus(5, 2))
Survivor: 2
>>> print(josephus(41, 3))
Survivor: 30
>>>

File: Task/Josephus-problem/Nim/josephus-problem-2.nim
func prisonerPos(n, k: Positive): int =
  ## The result is computed backwards. We start from the winner at
  ## position 0 on last round and compute its position on previous rounds.
  var pos = 0
  for i in 2..n:
    pos = (pos + k) mod i
  result = pos

echo "Survivor: ", prisonerPos(5, 2)
echo "Survivor: ", prisonerPos(41, 3)
